alfasim_sdk/status.py

```diff
@@ -0,0 +1,17 @@
+import attr
+from attr import attrib
+from attr.validators import instance_of
+
+from alfasim_sdk._validators import check_string_is_not_empty
+
+
+@attr.s(kw_only=True)
+class ErrorMessage:
+    model_name: str = attrib(validator=[instance_of(str), check_string_is_not_empty])
```

Comment:
`check_string_is_not_empty` should already use `instance_of(str)` behind the scenes, so users can simply use:

```python
model_name: str = attrib(validator=check_string_is_not_empty)
```